### DISCUSSION TITLE
logo change at login

### DISCUSSION THREAD

**Discussion by compnetmj** [YetiForceCompany/YetiForceCRM#17091] [How to] [answered]
Hello.
I'm trying to change the logo shown when logging in. 
I click the "Select" icon next to "Account", I select the file (the file name shows), click "Upload File".
I get the message "Sent successfully" and nothing - there is no logo.

Any suggestions ?

**Comment by mariuszkrzaczkowski** [comment]
https://doc.yetiforce.com/administrator-guides/faq/how-to-change-logo/#customize-the-login-screen-logo

**Reply by compnetmj** [reply]
Robiłem dokładnie wg wskazanej instrukcji - efekt jak wyżej czyli żaden.

**Reply by mariuszkrzaczkowski** [reply]
Sprawdz logi

**Reply by compnetmj** [reply]
Jak ? W Logi -> Podgląd logów mam Magento, Wapro ERP, Przełączanie użytkowników i jeszcze trzy ale żadnych logów z systemu do przejrzenia ...

**Comment by rskrzypczak** [comment]
I don't see much of a way for you to see the message "Sent successfully" and the logo hasn't changed.
Try reloading files into the browser cache (hard reset).

**Reply by compnetmj** [reply]
zrzut ekranu

https://github.com/YetiForceCompany/YetiForceCRM/assets/131910105/a9febfd5-b3aa-4a15-af34-e13d904940a8

**Reply by rskrzypczak** [reply, marked as answer]
After logging out, the logo should be visible on the login page.

**Reply by compnetmj** [reply]
Ok thank you very much - it worked :)
